.clang-format

```diff
@@ -4,7 +4,7 @@ BasedOnStyle: LLVM
 AccessModifierOffset: -8
 BreakBeforeBraces: Allman
 BreakConstructorInitializers: AfterColon
-ColumnLimit: 0
+ColumnLimit: 120
```

Comment:
this has no effect on our code, it seems a more reasonable value for developers that use it locally